core-kotlin-modules/pom.xml

```diff
@@ -52,6 +52,8 @@
         <module>core-kotlin-strings-3</module>
         <module>core-kotlin-files</module>
         <module>core-kotlin-operators</module>
+        <module>core-kotlin-arrays-2</module>
```

Comment:
The build fails with:
```
[ERROR] [ERROR] Some problems were encountered while processing the POMs:
[ERROR] Child module /Users/liam/git/kotlin-tutorials/core-kotlin-modules/core-kotlin-arrays-2 of /Users/liam/git/kotlin-tutorials/core-kotlin-modules/pom.xml does not exist @
```

Because this module does not exist. Remove this line to fix it.
```suggestion
```